h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -502,27 +534,55 @@ private void optionallySplitTrainingDataset() {
     // then we need to create one out of the original training set.
     if (!isCVEnabled()) {
       double[] splitRatios = null;
-      if (null == _validationFrame && null == _leaderboardFrame) {
-        splitRatios = new double[]{ 0.8, 0.1, 0.1 };
-        eventLog().info(Stage.DataImport,
-            "Since cross-validation is disabled, and none of validation frame and leaderboard frame were provided, " +
-                "automatically split the training data into training, validation and leaderboard frames in the ratio 80/10/10");
-      } else if (null == _validationFrame) {
-        splitRatios = new double[]{ 0.9, 0.1, 0 };
-        eventLog().info(Stage.DataImport,
-            "Since cross-validation is disabled, and no validation frame was provided, " +
-                "automatically split the training data into training and validation frames in the ratio 90/10");
-      } else if (null == _leaderboardFrame) {
-        splitRatios = new double[]{ 0.9, 0, 0.1 };
+      double validationRatio = null == _validationFrame ? 0.1 : 0;
+      double leaderboardRatio = null == _leaderboardFrame ? 0.1 : 0;
+      double blendingRatio = (_buildSpec.build_control.use_auto_blending && null == _blendingFrame) ? 0.2 : 0;
```

Comment:
@sebhrusen I agree, leaderboard can use validation metrics, even though they are slightly biased (since we use validation frame in early stopping).  The goal of the leaderboard is ranking, not unbiased test set set performance, and if the user wants that, they can pass a `leaderboard_frame` directly.